CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 ### Features
 
-* add more expressions to dynamic import parser ([#546](https://www.github.com/netlify/zip-it-and-ship-it/issues/546)) ([f29156e](https://www.github.com/netlify/zip-it-and-ship-it/commit/f29156ef5395795ad40e9df9556908b58aec6ae9))
+* add support for more expression types to dynamic import parser ([#546](https://www.github.com/netlify/zip-it-and-ship-it/issues/546)) ([f29156e](https://www.github.com/netlify/zip-it-and-ship-it/commit/f29156ef5395795ad40e9df9556908b58aec6ae9))
```

Comment:
This change was so that the E2E flow was triggered: https://github.com/netlify/zip-it-and-ship-it/pull/554/files#diff-d396261d6ce3045ca78e2b914f9b95deb0b675668f8f4866293b2e6451019bfcR7